### ISSUE TITLE
Standalone CDN library

### ISSUE THREAD

**Issue by dnitsch** [amido/stacks-terraform-azurerm#1]
**Is your feature request related to a problem? Please describe.**
Standalone CDN library for TF with flags to create the DNS for custom domain or just the profile.
could be useful when you want to have a profile managed centrally custom domains managed on a per deployment basis

**Describe the solution you'd like**
A library with required input and flags so that it can be used as part of core/sharedservices workflows as well as at a solution level

**Describe alternatives you've considered**

**Additional context**

**Comment by ElvenSpellmaker** [comment]
By CDN are you specifically referring to Azure CDN?

**Comment by dnitsch** [comment]
> By CDN are you specifically referring to Azure CDN?

yep, the module only creates a Microsofts own CDN implementation. up for discussion but it seems to be quite feature rich and manageable all through terraform
